src/dispatch-payload.ts

```diff
@@ -0,0 +1,62 @@
+import { LoadFunctionModule } from "./load-function-module.ts";
```

Comment:
The bulk of the logic in this file come from consolidating `mod.ts` and `local-run.ts`